src/rqt_image_view/image_view.cpp

```diff
@@ -75,9 +76,15 @@ void ImageView::initPlugin(qt_gui_cpp::PluginContext& context)
 
   connect(ui_.dynamic_range_check_box, SIGNAL(toggled(bool)), this, SLOT(onDynamicRange(bool)));
 
-  ui_.save_as_image_push_button->setIcon(QIcon::fromTheme("image-x-generic"));
+  ui_.save_as_image_push_button->setIcon(QIcon::fromTheme("document-save-as"));
   connect(ui_.save_as_image_push_button, SIGNAL(pressed()), this, SLOT(saveImage()));
 
+  std::string filepath = ros::package::getPath("rqt_image_view")+"/icons/grid_icon.png";
+  QPixmap pixmap(filepath.c_str());
+  ui_.grid_push_button->setIcon(QIcon(pixmap));
```

Comment:
Could we avoid adding this button? The value of `num_gridlines_` could be used for this: if it is zero it doesn't show a grid while it does for numbers > 0.

In that case the default should be `0` to match the current behavior of not showing the grid.